src/main/java/Glendon.java

```diff
@@ -0,0 +1,117 @@
+import java.util.Scanner;
+
+public class Glendon {
+    public static final int maxNumberOfTask = 100;
+    public static Task[] list = new Task[maxNumberOfTask];
```

Comment:
Array specifiers should be attached to the type not the variable. For example: int[] a = new int[20];